### PR TITLE
main/py-six: upgrade to 1.12.0, added tests

### PR DESCRIPTION
**Pull request by dimon222**
https://github.com/benjaminp/six/blob/master/CHANGES#L6

### CODE REVIEW

**Comment by dimon222** [comment]
Added tests.

File: main/py-six/APKBUILD [hunk 1]
```diff
@@ -20,6 +21,12 @@ build() {
 	python3 setup.py build
 }
 
+build() {
```
Comment (rnalrd) [inline main/py-six/APKBUILD]: I believe you meant to write check() ... :smile:

File: main/py-six/APKBUILD [hunk 2]
```diff
@@ -20,6 +21,12 @@ build() {
...
+	cd "$builddir"
```
Comment (rnalrd) [inline main/py-six/APKBUILD]: This can be omitted nowdays..

File: main/py-six/APKBUILD (hunk 1 above)
Comment (dimon222) [reply main/py-six/APKBUILD]: Eh, true.

Comment (dimon222) [reply main/py-six/APKBUILD]: Done, forcepushed new version.

File: main/py-six/APKBUILD (hunk 2 above)
Comment (dimon222) [reply main/py-six/APKBUILD]: Removed in new forcepush.

**Comment by rnalrd** [comment]
Merged

**Pull request closed without merging**